examples/example6/example6_withaxisymm.ipynb

```diff
@@ -686,34 +686,35 @@
     "        \"initial_dt\": DT_INITIAL,\n",
     "        \"adjust_dt\": adjust_dt,\n",
     "        \"time_precision\": 6,\n",
+    "        \"attempt_timestep_restart_on_divergence\": True\n",
```

Comment:
Note that allowing timestep restart on divergence was required for convergence in this example. We might want to look at this more closely down the line, something about using the 2D mesh with axisymmetry in this case made it more difficult to get this to converge.